src/services/emails/confirm-reset-password.js

```diff
@@ -1,17 +1,15 @@
 module.exports = (host, token) => {
-  let getTemplate = () => {
-    return {
-      subject: 'Style Point: Reset your password',
-      html: `
+  const getTemplate = () => ({
+    subject: 'Style Point: Reset your password',
+    html: `
         <p>
             You are receiving this because you (or someone else) have requested the reset of the password for your account.
             <br>Please click on the following link, or paste this into your browser to complete the process:<br><br>
-            http://${host}/api/confirm-reset-password/${token}<br><br>
+            ${host}?resetPasswordToken=${token}<br><br>
```

Comment:
yup. We're passing a template string with req.protocol [here](https://github.com/Ivan-Marquez/momentum/blob/hotfix/refactor/src/controllers/auth.controller.js#L62). Or we could just pass req as an argument and set the string concatenation inside the function.